guides/v2.2/reference/cli/magento-cloud.md

```diff
@@ -0,0 +1,22 @@
+---
+group: cli-reference
+title: magento-cloud
+functional_areas:
+  - Configuration
+  - System
+  - Setup
+---
+<!-- All the assigned and captured content is used in the included template -->
+{% assign tool = 'magento-cloud' %}
+
+{% assign file = site.data.codebase.cloud.magento-cloud %}
+
+{% assign edition = site.data.var.ece %}
+
+{% capture intro %}
+{: .bs-callout-info }
+You can call Magento CLI commands using shortcuts instead of the full command name. For example, you can call `{{ tool }} setup:upgrade` using `{{ tool }} s:up`, `{{ tool }} s:upg`, etc. See [shortcut syntax](https://symfony.com/doc/current/components/console/usage.html#shortcut-syntax) to understand how to use shortcuts with any Magento CLI command.
```

Comment:
So Magento Cloud CLI does not have the `setup:upgrade` call, but you can use the shortcut method in an interesting way. For example, `mgc ssh` opens a SSH to the remote environment instead of having to type the full `magento-cloud environment:ssh` command. Shortcuts are in parenthesis in the help and the second and third example of usage.

Reply:
I don't see any reason to keep this note then.

Reply:
Or create one for Cloud. I will take a closer look.